content/md/what-is/what-about-reference-entities.md

```diff
@@ -0,0 +1,39 @@
+---
+id: what-about-reference-entities
+themes: first-steps
+title: What about **reference entities**?
+popular: true
+ee-only: true
+related:
+---
+
+# What's the use case?
+
+Let's take an example of a product page.
```

Comment:
If you want to go to the next line, add two spaces at the end of your line. Otherwise, it won't work. ;)

```suggestion
Let's take an example of a product page.  
```